diff --git a/fiftyone/constants.py b/fiftyone/constants.py
index aed1b25..e7ca414 100644
--- a/fiftyone/constants.py
+++ b/fiftyone/constants.py
@@ -8,6 +8,7 @@ Package-wide constants.
 
 from datetime import datetime
 import os
+from shutil import which
 
 from packaging.version import Version
 
@@ -88,11 +89,7 @@ DEFAULT_COLOR_SCHEME = {
 }
 
 # MongoDB setup
-try:
-    from fiftyone.db import FIFTYONE_DB_BIN_DIR
-except ImportError:
-    # development installation
-    FIFTYONE_DB_BIN_DIR = os.path.join(FIFTYONE_CONFIG_DIR, "bin")
+FIFTYONE_DB_BIN_DIR = os.path.normpath(os.path.join(which("mongod"), '..'))
 
 DEFAULT_DB_DIR = os.path.join(FIFTYONE_CONFIG_DIR, "var", "lib", "mongo")
 MIGRATIONS_PATH = os.path.join(FIFTYONE_CONFIG_DIR, "migrations")
diff --git a/setup.py b/setup.py
index b8d0060..396eb08 100644
--- a/setup.py
+++ b/setup.py
@@ -46,7 +46,7 @@ INSTALL_REQUIRES = [
     # https://plotly.com/python/static-image-export/
     "kaleido!=0.2.1.post1",
     "matplotlib",
-    "mongoengine==0.24.2",
+    "mongoengine",
     "motor>=2.5",
     "numpy",
     "packaging",
@@ -67,14 +67,13 @@ INSTALL_REQUIRES = [
     "sseclient-py>=1.7.2,<2",
     "sse-starlette>=0.10.3,<1",
     "starlette>=0.24.0",
-    "strawberry-graphql==0.138.1",
+    "strawberry-graphql",
     "tabulate",
     "xmltodict",
     "universal-analytics-python3>=1.0.1,<2",
     "pydash",
     # internal packages
     "fiftyone-brain>=0.17.0,<0.18",
-    "fiftyone-db>=0.4,<2.0",
     "voxel51-eta>=0.12.7,<0.13",
 ]
 
