diff --git a/fiftyone/constants.py b/fiftyone/constants.py
index 1c08e3cfe..065d67bfe 100644
--- a/fiftyone/constants.py
+++ b/fiftyone/constants.py
@@ -7,6 +7,7 @@ Package-wide constants.
 """
 from datetime import datetime
 import os
+from shutil import which
 
 from packaging.version import Version
 
@@ -90,11 +91,7 @@ DEFAULT_COLOR_SCHEME = {
 }
 
 # MongoDB setup
-try:
-    from fiftyone.db import FIFTYONE_DB_BIN_DIR
-except ImportError:
-    # development installation
-    FIFTYONE_DB_BIN_DIR = os.path.join(FIFTYONE_CONFIG_DIR, "bin")
+FIFTYONE_DB_BIN_DIR = os.path.normpath(os.path.join(which("mongod"), '..'))
 
 DEFAULT_DB_DIR = os.path.join(FIFTYONE_CONFIG_DIR, "var", "lib", "mongo")
 MIGRATIONS_PATH = os.path.join(FIFTYONE_CONFIG_DIR, "migrations")
diff --git a/setup.py b/setup.py
index f8ac8e32f..c3e8ab462 100644
--- a/setup.py
+++ b/setup.py
@@ -50,7 +50,7 @@ INSTALL_REQUIRES = [
     # https://plotly.com/python/static-image-export/
     "kaleido!=0.2.1.post1",
     "matplotlib",
-    "mongoengine==0.24.2",
+    "mongoengine",
     "motor>=2.5",
     "numpy",
     "packaging",
@@ -71,13 +71,12 @@ INSTALL_REQUIRES = [
     "sseclient-py>=1.7.2,<2",
     "sse-starlette>=0.10.3,<1",
     "starlette>=0.24.0",
     "strawberry-graphql",
     "tabulate",
     "xmltodict",
     "universal-analytics-python3>=1.0.1,<2",
     # internal packages
     "fiftyone-brain>=0.17.0,<0.18",
-    "fiftyone-db>=0.4,<2.0",
     "voxel51-eta>=0.12.6,<0.13",
 ]
 
